src/components/back-link-helper/back-link-helper.js

```diff
@@ -17,10 +17,10 @@ BackLinkHelper.prototype.init = function init() {
     const invalidDomain = () => {
       const ref = this.document.referrer;
       const noRef = !ref;
-      const diffDom = ref.indexOf(this.window.location.host) === -1;
+      const diffDom = (ref) ? ref.indexOf(this.window.location.host) === -1 : false;
 
       // Allow PEGA domains
-      const allowList = ['maccount-np.hmrc.gov.uk', 'account.hmrc.gov.uk'];
+      const allowList = ['maccount-np.hmrc.gov.uk', 'account.hmrc.gov.uk','https://jira.tools.tax.service.gov.uk/'];
```

Comment:
dave just checking, why have we added jira in here? I don't know how private it is but this is an internal url

Reply:
I think I saw the catalogue appear before too

Reply:
Just using for testing logic in console, won't  be in final